src/main/java/mate/academy/internetshop/dao/jdbc/ProductDaoImpl.java

```diff
@@ -0,0 +1,115 @@
+package mate.academy.internetshop.dao.jdbc;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Optional;
+import mate.academy.internetshop.dao.ProductDao;
+import mate.academy.internetshop.lib.anno.Dao;
+import mate.academy.internetshop.model.Product;
+import mate.academy.internetshop.util.ConnectionUtil;
+import org.apache.log4j.Logger;
+
+@Dao
+public class ProductDaoImpl implements ProductDao {
+    private static final Logger LOGGER = Logger.getLogger(ConnectionUtil.class);
+
+    @Override
+    public Product create(Product element) {
+        String query = "INSERT INTO internetshop.products(name, price) VALUES (?, ?)";
+
+        try (Connection connection = ConnectionUtil.getConnection()) {
+            PreparedStatement preparedStatement = connection.prepareStatement(query);
+            preparedStatement.setString(1, element.getName());
+            preparedStatement.setDouble(2, element.getPrice());
+            preparedStatement.execute();
+        } catch (SQLException e) {
+            LOGGER.warn("Unable to execute create product query. Stack trace: " + e.getMessage());
+            throw new RuntimeException("Unable to execute create product query. Stack trace: " + e.getMessage());
```

Comment:
let's create custom DataProcessingException() and use it.
Also you can remove logger from catch block since our exception will be logged anyway